class.jetpack.php

```diff
@@ -829,7 +829,7 @@ function remove_non_jetpack_xmlrpc_methods( $methods ) {
 	 * from /xmlrpc.php so that we're replicating it as closely as possible.
 	 */
 	function alternate_xmlrpc() {
-		// phpcs:disable PHPCompatibility.PHP.RemovedGlobalVariables.http_raw_post_dataDeprecatedRemoved
```

Comment:
@jeherve I couldn't commit my changes without broadening `PHPCompatibility` exclusions in this file. Is that expected?

Reply:
Hm, I didn't have to do that at all. 🤔

Reply:
Makes me wonder if something changed with phpcs since then

Reply:
Hm. Could it be that you run a local version of `phpcs` with its own config, that may be different from the one in `vendor/bin/phpcs`, that ships with Jetpack?

Reply:
It doesn't look that way?
```
husky > pre-commit (node v10.15.3)
> composer install && vendor/bin/phpcs -p --runtime-set testVersion '5.2-' --standard=PHPCompatibilityWP --ignore=docker,tools,tests,node_modules,vendor --extensions=php 'class.jetpack.php'
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating autoload files
PHP CodeSniffer Config installed_paths set to ../../phpcompatibility/phpcompatibility-paragonie,../../phpcompatibility/phpcompatibility-wp,../../sirbrillig/phpcs-variable-analysis,../../wp-coding-standards/wpcs,../../phpcompatibility/php-compatibility
```

I'll just add them back manually

Reply:
We shouldn't need any changes to this file any more with our new approach.